docs_user/assemblies/assembly_migrating-databases-to-the-control-plane.adoc

```diff
@@ -4,7 +4,7 @@
 
 = Migrating databases to the control plane
 
-To begin creating the control plane, enable backend services and import the databases from your original {rhos_prev_long} {rhos_prev_ver} deployment.
+To begin creating the control plane, enable back-end services and import the databases from your original {rhos_prev_long} {rhos_prev_ver} deployment.
```

Comment:
Is this assembly/user story just the beginning (To begin creating the control plane) of the creation of the control plane? I'd like to see a complete picture (super procedure) somewhere of the list of user stories in order, that you must complete.